src/main/java/org/benetech/servicenet/adapter/anonymous/AnonymousDataAdapter.java

```diff
@@ -29,24 +30,27 @@
     private EntityManager em;
 
     @Override
-    public void importData(SingleImportData data) {
+    public DataImportReport importData(SingleImportData data) {
         Type listType = new TypeToken<ArrayList<RawData>>() {
         }.getType();
         List<RawData> entries = new Gson().fromJson(data.getSingleObjectData(), listType);
 
         AnonymousDataMapper mapper = AnonymousDataMapper.INSTANCE;
 
         //TODO: do not persist some entities if they already exists
+        //TODO: handle updates in reports as well
         for (RawData rawData : entries) {
             Location location = mapper.extractLocation(rawData);
             em.persist(location);
 
             Organization organization = mapper.extractOrganization(rawData)
-                .location(location).active(true).sourceDocument(data.getDocumentUpload());
+                .location(location).active(true).sourceDocument(data.getReport().getDocumentUpload());
```

Comment:
This class is going to be removed soon, as it probably will never be used - https://github.com/benetech/ServiceNet/issues/80
But I've updated it, as it wouldn't compile otherwise